docs/built_rst/tables/elements/street_segment.rst

```diff
@@ -57,6 +57,14 @@
 |                      |                           |              |              | **IncludesAllStreets** are true, this    |                                          |
 |                      |                           |              |              | value is ignored.                        |                                          |
 +----------------------+---------------------------+--------------+--------------+------------------------------------------+------------------------------------------+
+| HouseNumberPrefix    | ``xs:string``             | Optional     | Single       | Part of a street address. It may contain | If the field is not present of invalid,  |
```

Comment:
Same here. We need guidance on what these mean before adding them. See my comments/requests on #162.